jest-puppeteer.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+    launch: {
+        dumpio: true,
+        headless: true
+    },
+    browser: 'chromium',
```

Comment:
Can we run on ie/safari as well?

Reply:
Currently the puppeteer only supports Chrome and Firefox, but I think it will supports more browsers in the future. However, it currently can emulate different devices with different userAgents.